src/hdf5_back_gen.py

```diff
@@ -24,12 +24,14 @@ def resolve_unicode(item):
 with open(os.path.join(os.path.dirname(__file__), '..', 'share', 'dbtypes.json')) as f:
     RAW_TABLE = resolve_unicode(json.load(f))
 
+VERSION = ""
 TABLE_START = 0
+
 for row in range(0, len(RAW_TABLE)):
     current = tuple(RAW_TABLE[row])
-    if current[5] == "v1.3" and current[4] == "HDF5":
+    if current[4] == "HDF5" and current[5] > VERSION:
+        VERSION = current[5]
         TABLE_START = row
-        break
 
 V3_TABLE = list(tuple(row) for row in RAW_TABLE[TABLE_START:])
```

Comment:
I will add an upper bound to the RAW_TABLE range. In terms of this issue, I don't believe this is causing a problem since the code only targets types that are supported, the correct version, and tagged as HDF5.